Find_first_missing_positive_integer.rb

```diff
@@ -0,0 +1,33 @@
+=begin 
+This problem was asked by Stripe.
+
+Given an array of integers, find the first missing positive integer in linear time and constant space. 
+In other words, find the lowest positive integer that does not exist in the array. 
+The array can contain duplicates and negative numbers as well.
+
+For example, the input [3, 4, -1, 1] should give 2. The input [1, 2, 0] should give 3.
+
+You can modify the input array in-place.
+
+=end 
+
+def findfirstmissingpostiveint(array)

```

Comment:
we use snake case in ruby. 

`find_first_missing_positive_int`

Reply:
snake case?

Reply:
https://medium.com/better-programming/string-case-styles-camel-pascal-snake-and-kebab-case-981407998841